core/dbt/cli/utils.py

```diff
@@ -0,0 +1,32 @@
+from dbt.cli.flags import Flags
+from dbt.config import RuntimeConfig
+from dbt.config.runtime import Profile, Project, load_project, load_profile
+from dbt.flags import get_flags
+
+
+def get_profile(flags: Flags) -> Profile:
+    # TODO: Generalize safe access to flags.THREADS:
```

Comment:
It appears that the functions here are copied from `requires.py`. Please just consolidate it in one place